### PR TITLE
release v1.1.0

### PR DESCRIPTION
**Pull request by verinderpb**
**Description of changes:**

### Added

- AWS CodeCommit support for application patterns repository
- Support for Node.js version 18

### Fixed

- Updated the code to accommodate the new AWS S3 default security setting of disabling the ACL.

**Checklist**

- [x] :wave: I have run the unit tests, and all unit tests have passed.
- [ ] :warning: This pull request might incur a breaking change.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/cypress/fixtures/templaterepo/cdk/packages/compliant-dynamodb-table/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# `compliant-dynamodb-table`
+
+> TODO: description
```
Comment (gsingh04) [inline source/cypress/fixtures/templaterepo/cdk/packages/compliant-dynamodb-table/README.md:3]: TODO?

File: source/cypress/fixtures/templaterepo/cdk/packages/compliant-s3-bucket/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# `compliant-s3-bucket`
+
+> TODO: description
+
+## Usage
+
+```
+const compliantS3Bucket = require('compliant-s3-bucket');
+
+// TODO: DEMONSTRATE API
```
Comment (gsingh04) [inline source/cypress/fixtures/templaterepo/cdk/packages/compliant-s3-bucket/README.md:10]: TODO?

**Review by gsingh04 — APPROVED** [review]

File: source/cypress/fixtures/templaterepo/cdk/packages/compliant-dynamodb-table/README.md (hunk 1 above)
Comment (verinderpb) [reply source/cypress/fixtures/templaterepo/cdk/packages/compliant-dynamodb-table/README.md:3]: This is a template snippet for the users. The TODO is for users to add the required code to demonstrate the API.

File: source/cypress/fixtures/templaterepo/cdk/packages/compliant-s3-bucket/README.md (hunk 2 above)
Comment (verinderpb) [reply source/cypress/fixtures/templaterepo/cdk/packages/compliant-s3-bucket/README.md:10]: This is a template snippet for the users. The TODO is for users to add the required code to demonstrate the API.

**Review by van-vothanh — APPROVED** [review]

**Pull request merged**